doc/source/ray-core/actors.rst

```diff
@@ -466,6 +466,23 @@ define can run. This also implies that tasks are scheduled more flexibly,
 and that if you don't need the stateful part of an actor, you're mostly
 better off using tasks.
 
+Task Events 
+-----------
+
+By default, Ray will trace the execution of actor tasks, reporting task status events and profiling events
+that's used by Ray Dashboard and :ref:`State API <state-api-overview-ref>`.
+
+You could disable task events for the actor by setting the `enable_task_events` option to `False` in :func:`ray.remote() <ray.remote>` and :meth:`.options() <ray.actor.ActorClass.options>`, which would reduce the overhead of task execution, and also reduce the amount of data sent to the Ray Dashboard
```

Comment:
```suggestion
You can disable task events for the actor by setting the `enable_task_events` option to `False` in :func:`ray.remote() <ray.remote>` and :meth:`.options() <ray.actor.ActorClass.options>`, which reduces the overhead of task execution, and the amount of data the actor sends to the Ray Dashboard.
```

Reply:
Not sure if it's the actor that sends the data, or Ray. Please correct the subject as needed.